### PR TITLE
Add shortcut to switching to sovereign cloud within the tenant view 

### PR DESCRIPTION
**Pull request by motm32**
This allows users to quickly change their sovereign cloud setting without having to go into the user settings. 

The shortcut lives here under the gear icon: 
image

There is also a command palette command titiled: `Azure: Configure Sovereign Cloud`

This brings up the below quick pick. The names correspond with the names on the setting.  When an item is selected the setting `microsoft-sovereign-cloud.environment` is changed to the corresponding value.

(I included the generic azure one so user could switch back easily)

image

I tested this by switching between an Azure China account and a default Azure account and had no issues.

### CODE REVIEW

**Comment by alexweininger** [comment]
does this bring up a quick pick or something

**Comment by motm32** [comment]
> Can you put a screenshot of the quick pick please

Yup I'll edit the description to add that

**Comment by alexweininger** [comment]
just some quick feedback: Maybe we can put (Default) in the description of the "Azure" normal option?

**Comment by motm32** [comment]
> just some quick feedback: Maybe we can put (Default) in the description of the "Azure" normal option?

Yup I'll do that!

**Comment by alexweininger** [comment]
Also I think we'll merge this after we merge the tenant view PR. There might be other small features like this on top of the tenant view that we want to add.

**Comment by motm32** [comment]
> Also I think we'll merge this after we merge the tenant view PR. There might be other small features like this on top of the tenant view that we want to add.

Sounds good I'll change the branch to main before we merge this.

File: src/commands/sovereignCloud/SovereignCloudListStep.ts
Comment (alexweininger) [inline src/commands/sovereignCloud/SovereignCloudListStep.ts:15]: If they pick custom, should we take them to the setting page so they can actually configure it? We can't make this change later though.

Comment (motm32) [reply src/commands/sovereignCloud/SovereignCloudListStep.ts:15]: Yah we should change the custom since it will cause an error at the moment

**Review by alexweininger — APPROVED** [review]

**Review by alexweininger — APPROVED** [review]

**Review by alexweininger — APPROVED** [review]

**Pull request merged**
